src/test/java/org/broadinstitute/consent/http/service/DataAccessRequestServiceTest.java

```diff
@@ -1,5 +1,7 @@
 package org.broadinstitute.consent.http.service;
 
+import org.broadinstitute.consent.http.enumeration.UserRoles;
+import org.broadinstitute.consent.http.models.*;
```

Comment:
Avoid star imports

Reply:
See https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports- for how to do this in your IDE